package.json

```diff
@@ -9,6 +9,7 @@
     "migrate": "knex migrate:latest --knexfile config/knexfile.js",
     "rollback": "knex migrate:rollback --knexfile config/knexfile.js",
     "watch:dev": "nodemon",
+    "server": "supervisor server.js",
```

Comment:
Did you mean to push this as part of the commit? If so we should probably rename the script to be more easily understood. Something like `watch:dev:windows` since this is a script for running the server on windows machines.

Reply:
I updated the script name to your suggestion `watch:dev:windows` incase any other windows users are having issues with nodemon.